tests/python_tests/test_tokenizer.py

```diff
@@ -18,6 +19,21 @@
 )
 
 
+def load_hf_tokenizer(model_id: str, hf_load_params: dict = None):
+    hf_load_params = hf_load_params or {}
+    return AutoTokenizer.from_pretrained(model_id, **hf_load_params)
+
+
+def convert_and_load_genai_tokenizer(hf_tokenizer : AutoTokenizer, models_path: Path):
+    from openvino_tokenizers import convert_tokenizer
+    from openvino import save_model
+
+    tokenizer, detokenizer = convert_tokenizer(hf_tokenizer, with_detokenizer=True)
+    save_model(tokenizer, models_path / "openvino_tokenizer.xml")
+    save_model(detokenizer, models_path / "openvino_detokenizer.xml")
+    return openvino_genai.Tokenizer(models_path)
+
```

Comment:
please, move these 2 functions to common.py and reuse here https://github.com/openvinotoolkit/openvino.genai/blob/master/tests/python_tests/common.py#L449-L470

Reply:
done

Reply:
please, read my comment once again

Reply:
I see. I indeed didn't fully understood it the first time. I will reuse convert tokenizer there